### DISCUSSION TITLE
Join NumFOCUS as an affiliated project

### DISCUSSION THREAD

**Discussion by cpcloud** [ibis-project/ibis#4158] [Polls]
I think we should apply to NumFOCUS as an [affiliated project](https://numfocus.org/sponsored-projects/affiliated-projects) (as opposed to fiscal sponsorship). This will give the project more visibility for the target user base.

Thoughts?

**Comment by jreback** [comment]
+1

**Comment by cpcloud** [comment]
We've applied, and are waiting to hear back!

**Comment by cpcloud** [comment]
We are now an affiliated project!
